Products/CMFPlone/resources/browser/resource.py

```diff
@@ -25,6 +25,13 @@
 }
 
 
+def is_external_url(resource):
+    # we check if the resource string starts with http and //
+    # according to relative URI definition in
+    # https://www.ietf.org/rfc/rfc3986.txt chapter 4.2
+    return resource.startswith("http") or resource.startswith("//")
```

Comment:
These are just suggestion, do not consider them as blockers :)

I do not know much about what's happening here, but checking the number of times `external` is used while initializing the `Plone*Resource` it seems to me the check should be done in the `__init__` of that class.

BTW I would not add this helper has a module function.
I would very much prefer to have it as a class method, even a static one.

Reply:
Thanks for you suggestion. I will change this.

We have a migrated theme here with "//foo.bar/foobar.js" URLs for external resources. I originally wanted to replace them, but then I noticed the traceback in the resource registry which breaks the whole site. This is because the viewletmanager tries to traverse to this "path".

So instead of catching the exception here https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/resources/utils.py#L62 I'd rather support this notation.

Reply:
Yeah, it is for sure something we should support :+1: